CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 ## v1.4.0
 
 - [core] fixed handling of environment variables on Windows [#7973](https://github.com/eclipse-theia/theia/pull/7973)
+- [core] added functionality for handling context menu for `tab-bars` without changing focus.[#6965](https://github.com/eclipse-theia/theia/pull/6965)
+
+Breaking Changes:
+
+- [core] Context-menu for `tab-bars` requires an `Event` to be passed onto it to perform actions without changing focus [#6965](https://github.com/eclipse-theia/theia/pull/6965)
+  - Removed the logic of giving focus to the widget upon opening the context menu from `handleContextMenuEvent()` instead functionality added to handle it without focus
+  - This change prevents the context menu to trigger an activation for a widget
```

Comment:
I don't think the change _prevents_ the activation, rather that the change triggers the context-menu for a given shell tabbar without the need to activate it. We no longer activate a tabbar when triggering the context-menu if the _event_ is passed.